src/engine/input/InputManager.cs

```diff
@@ -16,6 +16,12 @@ public class InputManager : Node
 {
     private static InputManager staticInstance;
 
+    /// <summary>
+    /// A bool indicating whether a rebinding is in progress
```

Comment:
XML element contents should be intended by 2 spaces.

Reply:
Aha, so just that I get this. This is going to be exported to some xml(-like) document and indenting here makes sure that the formatting in that document is already okay?

Reply:
I think you can tell a C# compiler to export the documentation, but we don't do that. However the style guide says (at least it should) that XML comments are indented for easier reading and style consistency. Once jetbrains get their stuff together, I'll enable an automatic check for this.